aws/scripts/sh/create_eventbridge_rule.sh

```
@@ -0,0 +1,74 @@
+        \"InputTemplate\": \"{ \\\"bucket_name\\\": \\\"$bucket_name\\\" }\"
+      }
+  }]"
+
+echo "Lambda added as target in EventBridge rule"
+
+# Check if the permission for EventBridge to invoke Lambda already exists
+existing_permission=$(aws lambda get-policy --function-name s3-cleanup-lambda 2>/dev/null | grep "$rule_name" || true)
+
+if [ -z "$existing_permission" ]; then
+  echo "🔒 Granting permission for EventBridge to invoke Lambda..."
+  statement_id="AllowEventBridgeInvoke-${sanitized_rule_name}-$(date +%s)"
```

Shellcheck (0.10.0)

[warning] sanitized_rule_name is referenced but not assigned.

(SC2154)